src/containers/MyNdla/components/MyNdlaBreadcrumb.tsx

```diff
@@ -40,12 +39,11 @@ const MyNdlaBreadcrumb = ({ breadcrumbs, page }: Props) => {
   const { t } = useTranslation();
 
   const baseCrumb = types[page];
-  const crumbs = [{ to: baseCrumb.to, name: t(baseCrumb.name) }].concat(
-    breadcrumbs.map((bc) => ({
-      name: bc.name,
-      to: `/minndla/${page}/${bc.id}`,
-    })),
-  );
+  const crumbs = breadcrumbs.map((bc) => ({
+    name: bc.name,
+    to: `/minndla/${page}/${bc.to}`,
+  }));
+  crumbs.unshift({ to: baseCrumb.to, name: t(baseCrumb.name) });
```

Comment:
Mulig å gjøre dette på en mer immutable måte? `[{to: baseCrumb.to, name: t(baseCrumb.name)}].concat(breadcrumbs.map(...))`